core/spec/models/spree/image_spec.rb

```diff
@@ -9,6 +9,18 @@
     let(:default_style) { :product }
   end
 
+  it 'is valid when attachment has allowed content type' do
+    image = build(:image,
+                  attachment: File.open(File.join('lib', 'spree', 'testing_support', 'fixtures', 'blank.jpg')))
```

Comment:
I like to put a blank line between Given/Then/When contexts in the specs... However, I think this is not consistently used in the codebase, so feel free to ignore the comment

Reply:
I am not noticing this in the rest of the examples, I think I will leave it as is to conform for the time being. Thanks for the suggestion!